common.yml

```diff
@@ -1,13 +1,11 @@
+
 service:
```

Comment:
This naming pre-dates this but "service" seems like a bit of a odd name for a service. Maybe more verbosity here wouldn't hurt because having a service called service reads odd imho.